theme/static/js/hs_file_browser.js

```diff
@@ -1192,7 +1192,7 @@ function showFileTypeMetadata(file_type_time_series, url){
                   setupModelProgramFileTypeUI();
                   setupModelProgramTypeUI();
                   $("#mi-json-schema-file").change(function () {
-                    $(".btn-form-submit").show();
+                    $("#filetype-modelprogram").find(".btn-form-submit").show();
```

Comment:
this [get_html_forms()](https://github.com/hydroshare/hydroshare/blob/master/hs_file_types/models/model_program.py#L293) is so interesting! I'm curious why we used [dominate](https://github.com/Knio/dominate) historically, instead of a django template--maybe the logic gets too complicated? Anyway, just curious...I'll have to take a deeper look. But changes in this PR look good!

Reply:
When I started using **dominate** in hydroshare. I found easier for me to create complex form/html content using dominate compared to creating html template file. I guess it would not be true for a frontend developer.

Reply:
Cool! I haven't used dominate--seems nice and clean and well done!